From: Lucas Stach <l.stach@pengutronix.de>
Date: Mon, 6 Nov 2017 15:48:54 +0100
Subject: [PATCH] ARM: dts: imx6: add brown-out detection configuration

This enables brown-out detection on the anatop digital regulators
with the default offset of 50mV that is used in the standard OPPs
between minimal and typical voltage.

Signed-off-by: Lucas Stach <l.stach@pengutronix.de>
---
 arch/arm/boot/dts/imx6qdl.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl.dtsi b/arch/arm/boot/dts/imx6qdl.dtsi
index c003e62bf290..ec1660dd08a2 100644
--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -759,9 +759,11 @@
 					anatop-delay-reg-offset = <0x170>;
 					anatop-delay-bit-shift = <24>;
 					anatop-delay-bit-width = <2>;
+					anatop-bo-shift = <0>;
 					anatop-min-bit-val = <1>;
 					anatop-min-voltage = <725000>;
 					anatop-max-voltage = <1450000>;
+					anatop-bo-offset-microvolt = <50000>;
 				};
 
 				reg_pu: regulator-vddpu@20c8140 {
@@ -777,9 +779,11 @@
 					anatop-delay-reg-offset = <0x170>;
 					anatop-delay-bit-shift = <26>;
 					anatop-delay-bit-width = <2>;
+					anatop-bo-shift = <8>;
 					anatop-min-bit-val = <1>;
 					anatop-min-voltage = <725000>;
 					anatop-max-voltage = <1450000>;
+					anatop-bo-offset-microvolt = <50000>;
 				};
 
 				reg_soc: regulator-vddsoc@20c8140 {
@@ -795,9 +799,11 @@
 					anatop-delay-reg-offset = <0x170>;
 					anatop-delay-bit-shift = <28>;
 					anatop-delay-bit-width = <2>;
+					anatop-bo-shift = <16>;
 					anatop-min-bit-val = <1>;
 					anatop-min-voltage = <725000>;
 					anatop-max-voltage = <1450000>;
+					anatop-bo-offset-microvolt = <50000>;
 				};
 			};
 
